src/compresswindow.cpp

```diff
@@ -4,7 +4,7 @@
 #include <QFileDialog>
 
 CompressWindow::CompressWindow(QWidget *parent) :
-    QMainWindow(parent),
+    QDialog(parent),
```

Comment:
Is it good use Dialog instead of MainWindow? I don't think this will good UX side. Previously I already turn this into dialog, but I'm not fit to use it.

Reply:
Yes, your application Should have only one MainWindow. Plus this acts like a dialog so why not start with QDialog

Reply:
Ok, thanks.